CMSIS/NN/Source/BasicMathFunctions/arm_elementwise_mul_s8.c

```diff
@@ -21,8 +21,8 @@
  * Title:        arm_elementwise_mul_s8
  * Description:  Element wise multiplication
  *
- * $Date:        December 16 2019
```

Comment:
For the changes in files from 2019, please update the copyright as well from 2010-2019 to 2010-2020..